tools/geospatial.go

```diff
@@ -456,6 +560,50 @@ func GetGeospatialData(gd *GeoData, fs filestore.FileStore, geomFilePath string,
 			f.Banks = append(f.Banks, bankLayers...)
 			log.Println("Extracted banks and cross-sections")
 
+		case strings.HasPrefix(line, "BreakLine Name="):
+			blLayer, err := getBreakLine(sc, transform)
+			switch {
+			case err != nil:
+				switch {
+				case err.Error() == "Invalid Line Geometry":
+					log.Println("Skipped", blLayer.FeatureName, err.Error())
+				default:
+					return err
+				}
+			default:
+				f.BreakLines = append(f.BreakLines, blLayer)
+				log.Println("Extracted break line")
+			}
+
+		case strings.HasPrefix(line, "BC Line Name="):
+			bcLayer, err := getBCLine(sc, transform)
+			switch {
+			case err != nil:
+				switch {
+				case err.Error() == "Invalid Line Geometry":
+					log.Println("Skipped", bcLayer.FeatureName, err.Error())
+				default:
+					return err
+				}
+			default:
+				f.BCLines = append(f.BCLines, bcLayer)
+				log.Println("Extracted boundary conditions line")
+			}
+
+		case strings.HasPrefix(line, "Connection="):
```

Comment:
Same comment as above about adding a "Field"

Reply:
- [ ] See if we want any field in connections.